skimage/exposure/tests/test_exposure.py

```diff
@@ -115,6 +115,23 @@ def test_normalize():
     assert_equal(frequencies, expected)
 
 
+# Test multichannel histograms
+# ============================
+
+def test_multichannel_hist_common_bins_uint8():
+    """Check that all channels use the same binning."""
+    # First channel contains zeros
+    # Second channel contains 255
+    im = np.array((np.zeros((5, 5)), np.ones((5, 5))*255), dtype=np.uint8)
+    im = np.moveaxis(im, source=0, destination=2)
```

Comment:
WOW, `moveaxis` is so much more intuitive when you include the keyword arguments! :clap:

Reply:
I'm quoting numpy:
> save those O(100) nanoseconds!
> umr_maximum = um.maximum.reduce

https://github.com/numpy/numpy/blob/master/numpy/core/_methods.py#L16